SVG/System76.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1024 1024">
+   <circle id="Background" cx="512" cy="512" r="512" style="fill:#595149"/>
```

Comment:
- Remove `id="Background"`
- Change the `style="fill:#595149"` ➜ `style="fill:#574f4a"`